/**
 * Forcefully throws a failed assertion.
 *
 * @example
 * ```ts
 * import { fail } from "@std/assert/fail";
 *
 * fail("Deliberately failed!"); // Throws
 * ```
 */ export declare function fail(msg?: string): never;
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImZhaWwudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBSUE7Ozs7Ozs7OztDQVNDLEdBQ0QsT0FBTyxpQkFBUyxLQUFLLE1BQU0sTUFBTSxHQUFHLEtBQUsifQ==